FitNesseRoot/DbFit/AcceptanceTests/CoreTests/StoredProcNoParams/content.txt

```diff
@@ -1,5 +1,9 @@
 !3 If there are no parameters, Execute Procedure needs just one row
 
+|Clean|
+|table|clean?|
+|users|true|
```

Comment:
That perhaps means that some other tests don't perform tear down at the end. Do we know which ones?